examples/pseudopotentials.jl

```diff
@@ -77,49 +77,77 @@ end
 # We load the HGH and UPF PSPs using `load_psp`, which determines the
 # file format using the file extension.
 
-psp_hgh = load_psp("hgh/lda/mg-q2.hgh");
+psp_hgh = load_psp("hgh/lda/li-q3.hgh");
 psp_upf = load_psp(download_gunzip_upf(URL_UPF));
 
 # Next, we define some parameters for the energy cutoff convergence.
 # These are fairly lose parameters; in practice tighter parameters
 # and higher cutoffs might be needed. The PseudoDojo provides recommended
 # cutoffs for its pseudopotentials as notes on the periodic table.
 # Ecuts = 28:4:64
-Ecuts = 52:4:76
-tol   = 1e-4
+Ecuts = 20:4:68
+tol   = 1e-2
 
 conv_hgh = converge_Ecut(Ecuts, psp_hgh, tol)
 conv_upf = converge_Ecut(Ecuts, psp_upf, tol)
 
-function run_bands(Ecut, psp)
-    a = -1.6015138650u"Å"
-    b = -2.7739033832u"Å"
-    c = -5.1266910000u"Å"
+# From this small convergence study, we can look at the difference in convergence w.r.t.
+# `Ecut` between the analytical and numeric pseudos. We see that the HGH pseudopotential
+# is much *harder*, i.e. it requires a higher energy cutoff, than the UPF PSP. In general,
+# numeric pseudopotentials tend to be softer than analytical pseudos because of the
+# flexibility of sampling an arbitrary functions on a grid. 
+
+begin
+    plt = plot(xlabel="Ecut [Ha]", ylabel="Error [Ha]")
+    plot!(plt, conv_hgh.Ecuts, conv_hgh.errors, label="HGH")
+    plot!(plt, conv_upf.Ecuts, conv_upf.errors, label="PseudoDojo UPF")
+    hline!(plt, [tol], label="tol", color=:grey, linestyle=:dash)
+end
+
+#md # ```@raw html
+#md # <img src="../../assets/Li_convergence.png" width=600 height=400 />
+#md # ```
+#nb # <img src="https://docs.dftk.org/stable/assets/Li_convergence.png" width=600 height=400 />
```

Comment:
You only need to include the images if they go beyond the Ecut range you have specified in the docs. Otherwise the docs should generate (and include and show) them automatically if the last line of code is returning a plot.

Reply:
I'm going to cut the convergence code and just show the static plots. I'll leave in the bandstructures at loose parameters, and if they're very bad, I'll also include static plots.

Reply:
Sounds good :+1: